doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,346 @@
+    "description": "The AIO requirements to run this connector",
+    "properties": {
+      "aioMinVersion": {
+        "type": "string",
+        "description": "The minimal (inclusive) version that can run this connector."
+      },
+      "aioMaxVersion": {
+        "type": "string",
+        "description": "The maximal (inclusive) version that can run this connector."
+      }
+    }
+  },
+  "connectorConfigurationSchema": {
+    "description": "The connector-level JSON schemas for the 'additional configuration' field. A JSON schema must be only if that field is used by this connector.",
+    "$ref": "http://json-schema.org/draft-07/schema#"
+  },
+  "inboundEndpoints": {
+    "type": "array",
+    "description": "The array of inbound endpoints in a device that this connector will be deployed for.",
+    "items": [
+      {
+        "type": "object",
+        "properties": {
+          "endpointType": {
+            "type": "string",
+            "description": "The type of this inbound endpoint including the version like 'Microsoft.Media:1.2.0'"
+          },
+          "asset": {
+            "type": "object",
+            "description": "The array of assets that reference this inbound endpoint",
+            "properties": {
+              "configurationSchemas": {
+                "type": "object",
+                "description": "The JSON schemas for the asset-level stringified JSON fields. A JSON schema must be provided for each field only if that field is used by this connector.",
+                "properties": {
+                  "datasetConfigurationSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  },
+                  "datasetDataPointConfigurationSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  },
+                  "eventConfigurationSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  },
+                  "eventDataPointConfigSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  },
+                  "streamConfigurationSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  },
+                  "managementGroupConfigurationSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  },
+                  "actionConfigurationSchema": {
+                    "$ref": "http://json-schema.org/draft-07/schema#"
+                  }
+                }
+              },
+              "supportedEventDestinations": {
+                "type": "array",
+                "description": "The array of destinations that this asset's events supports.",
+                "items": {
+                  "type": "string",
+                  "enum": ["Mqtt", "BrokerStateStore", "Storage"]
+                }
+              },
+              "defaultEventDestinations": {
```

Comment:
Is this an array or a single default destination...

Reply:
Should be just one. I will make this not an array